addons/dragging/CfgEventHandlers.hpp

```diff
@@ -32,6 +32,16 @@ class Extended_Init_EventHandlers {
             init = QUOTE(_this call DFUNC(initObject));
         };
     };
+    class RoadCone_F {
+        class ADDON {
+            init = QUOTE(_this call DFUNC(initObject));
+        };
+    };
+    class RoadCone_L_F {
```

Comment:
"RoadCone_F" and "RoadCone_L_F" is not part of "ReammoBox_F" its a part of "thingX".